src/sentry/integrations/slack/utils.py

```diff
@@ -129,14 +139,8 @@ def build_attachment(group, event=None, tags=None, identity=None, actions=None,
         actions = []
 
     try:
-        assignee = GroupAssignee.objects.get(group=group).user
-        assignee = {
-            'text': assignee.get_display_name(),
-            'value': assignee.username,
-        }
-
-        # Add unassign option to the top of the list
-        assignees.insert(0, UNASSIGN_OPTION)
+        assignee = GroupAssignee.objects.get(group=group).assigned_actor()
+        assignee = format_actor_option(assignee)
```

Comment:
... maybe do this on one line? maybe not

Reply:
IMO it's fine